### DISCUSSION TITLE
Add 'custom_family' to the model info

### DISCUSSION THREAD

**Discussion by GidonFrischkorn** [venpopov/bmm#181] [Ideas]
I wondered if it makes sense to add a 'True/False' indicator to the model info that specifies if the model uses a custom
Family or is configured using the non-linear formula syntax. That would probably help in generalizing some of the processing across different model implementations (e.g. applying link functions to model parameters). What do you think, @venpopov?

**Comment by venpopov** [comment]
If it proves helpful, we could add it, why not. But also we should try to not add unnecessary information, because it adds burden for maintenance and more instructions for developers. So I'd suggest to wait for a clear case where it does help- maybe you already have that when working with the M3?

At least for the current models it is not necessary. In the PR I just posted (#152), I generalized quite a bit of how to formula syntax is constructed, and so far the processing is the same regardless of whether a non-linear formula syntax is used - this is recognized by the methods, so not necessary to have an explicit tag. 

But if you think it will be clearly needed, we can add them when the need arises.
